mesh/pmesh.hpp

```diff
@@ -47,12 +47,22 @@ class ParMesh : public Mesh
    /// Shared objects in each group.
    Table group_svert;
    Table group_sedge;
-   Table group_sface;
+   Table group_stria;  // contains shared triangle indices
```

Comment:
(This will make merging Bob's `cut-mesh-groups-dev` a real pain, but I guess it's better to merge this first and redo Bob's refactoring that the other way around).